### PR TITLE
Create table

### PR DESCRIPTION
**Pull request by paulinagonzalezc**
## Vet clinic database: create animals table

In this part of the project, I completed the following requirements:✨
* Create a database named vet_clinic
* Create a table named animals with the indicated columns
* Insert the indicated data
* Write the required queries

Here I share the screenshots with the results of the queries: 

### Schema
Screenshot 2023-01-17 at 17 36 10

## Data
Screenshot 2023-01-17 at 17 51 52

## Queries
* Find all animals whose name ends in "mon".
Screenshot 2023-01-17 at 17 57 49

* List the name of all animals born between 2016 and 2019.
Screenshot 2023-01-17 at 18 42 32

* List the name of all animals that are neutered and have less than 3 escape attempts.
Screenshot 2023-01-17 at 18 42 58

* List the date of birth of all animals named either "Agumon" or "Pikachu".
Screenshot 2023-01-17 at 18 05 21

* List name and escape attempts of animals that weigh more than 10.5kg
Screenshot 2023-01-17 at 18 06 51

* Find all animals that are neutered.
Screenshot 2023-01-17 at 18 09 17

* Find all animals not named Gabumon.
Screenshot 2023-01-17 at 18 10 39

* Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
Screenshot 2023-01-17 at 18 13 23

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT name from animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name from animals WHERE neutered = 'true' AND escape_attempts < 3;
+SELECT date_of_birth from animals WHERE name = 'Agumon' OR name = 'Pikachu';
+SELECT name and escape_attempts from animals WHERE weight_kg > 10.5;
+SELECT * from animals WHERE neutered IS 'true';
+SELECT * from animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
```
Comment (Ismailco) [inline queries.sql]: - [x] You should add a query to Find all animals not named Gabumon, kindly make sure to implement this so all the quests are included

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @paulinagonzalezc, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (paulinagonzalezc) [reply queries.sql]: Thank you @Ismailco , I will correct this!

**Review by Reem-lab — APPROVED** [review]
# Hi @paulinagonzalezc, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Followed Gitflow ✅
- Great working on the functionlaity ✅
- Descriptive PR ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,106 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (Reem-lab) [inline README.md:104]: > * [OPTIONAL] I suggest adding the `MIT.md` file to your project :ok_hand:, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it :wink:
>
> > The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. :100:
>

**Review by Reem-lab — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (paulinagonzalezc) [reply README.md:104]: Thank you @Reem-lab! I will add the file!

**Pull request merged**